qtodotxt/lib/tasklib.py

```diff
@@ -44,6 +44,11 @@ def _reset(self):
         self.due_error = ""
         self.threshold = None
         self.keywords = {}
+        self.recMode = None
+        self.recModeCompl = 0   # Original due date mode: Task recurs from original due date
+        self.recModeOrDue = 1   # Completion date mode: Task recurs from completion date
```

Comment:
As Far as I understand you should create an Enum called RecursiveMode. Not 2 class variables